operators/ejector/ejector.go

```diff
@@ -116,6 +116,14 @@ func (e *Ejector) Eject(ctx context.Context, nonsignerMetrics []*NonSignerMetric
 		}
 	}
 
+	if len(nonsigners) == 0 {
+		e.logger.Info("No operators to eject")
+		e.metrics.IncrementEjectionRequest(mode, codes.OK)
+		return &EjectionResponse{
+			TransactionHash: "",
```

Comment:
can you add a comment to EjectionResponse that empty txn hash (when Eject has no error) means no nonsigners to eject?